spec.html

```diff
@@ -2805,32 +2805,64 @@
-      <p>The following abstract operations are used in this specification to access the components of references:</p>
-      <ul>
-        <li>
-          <dfn aoid="GetBase" id="ao-getbase">GetBase</dfn>(_V_). Returns the base value component of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="GetReferencedName" id="ao-getreferencedname">GetReferencedName</dfn>(_V_). Returns the referenced name component of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="IsStrictReference" id="ao-isstrictreference">IsStrictReference</dfn>(_V_). Returns the strict reference flag of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="HasPrimitiveBase" id="ao-hasprimitivebase">HasPrimitiveBase</dfn>(_V_). Returns *true* if Type(_V_'s base value component) is Boolean, String, Symbol, or Number; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsPropertyReference" id="ao-ispropertyreference">IsPropertyReference</dfn>(_V_). Returns *true* if either the base value component of the reference _V_ is an object or HasPrimitiveBase(_V_) is *true*; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsUnresolvableReference" id="ao-isunresolvablereference">IsUnresolvableReference</dfn>(_V_). Returns *true* if the base value component of the reference _V_ is *undefined*; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsSuperReference" id="ao-issuperreference">IsSuperReference</dfn>(_V_). Returns *true* if the reference _V_ has a thisValue component; otherwise returns *false*.
-        </li>
-      </ul>
       <p>The following abstract operations are used in this specification to operate on references:</p>
 
+      <emu-clause id="sec-getbase" aoid="GetBase">
+        <h1 id="ao-getbase">GetBase ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the base value component of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-getreferencedname" aoid="GetReferencedName">
+        <h1 id="ao-getreferencedname">GetReferencedName ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the referenced name component of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-isstrictreference" aoid="IsStrictReference">
+        <h1 id="ao-isstrictreference">IsStrictReference ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. Return the strict reference flag of _V_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-hasprimitivebase" aoid="HasPrimitiveBase">
+        <h1 id="ao-hasprimitivebase">HasPrimitiveBase ( _V_ )</h1>
+        <emu-alg>
+          1. Assert: Type(_V_) is Reference.
+          1. If Type(_V_'s base value component) is Boolean, String, Symbol, or Number, return *true*; otherwise return *false*.
```

Comment:
(a) Not according to past versions of the spec.
(b) No, because `null` fails RequireObjectCoercible?
(c) Beats me, this is just an editorial PR.